provider/build.gradle

```diff
@@ -17,28 +17,41 @@ android {
             minifyEnabled false
         }
         release {
-            debuggable false
+            debuggable true
```

Comment:
Should this be false?

Reply:
FML I can't remember why I changed it back then. I've set it to false for release builds for nor